server/routes/session.js

```diff
@@ -1,15 +1,15 @@
 export default (app) => {
   app
     .get('/session/new', async (req, reply) => {
-      await reply.render('session/new', { data: {}, errors: {} });
+      await reply.render('session/new', { data: { user: { email: '', password: '' } }, errors: {} });
```

Comment:
const data = new objection.models.user()? so we won't hardcode properties

Reply:
done